packages/ozone/src/mod-service/index.ts

```diff
@@ -559,27 +559,17 @@ export class ModerationService {
         for (const cid of blobCids) {
           blobValues.push({
             eventType,
+            takedownRef,
             subjectDid: subject.did,
+            subjectUri: subject.uri || null,
             subjectBlobCid: cid.toString(),
-            takedownRef,
           })
         }
       }
-      const blobEvts = await this.db.db
-        .insertInto('blob_push_event')
-        .values(blobValues)
-        .onConflict((oc) =>
-          oc
-            .columns(['subjectDid', 'subjectBlobCid', 'eventType'])
-            .doUpdateSet({
-              takedownRef,
-              confirmedAt: null,
-              attempts: 0,
-              lastAttempted: null,
-            }),
-        )
-        .returning(['id', 'subjectDid', 'subjectBlobCid', 'eventType'])
-        .execute()
+      const blobEvts = await this.eventPusher.logBlobPushEvent(
```

Comment:
This is unrelated to the current implementation but felt nice to keep this refactoring.